### DISCUSSION TITLE
I can't apply the SDK, it always reports an error

### DISCUSSION THREAD

**Discussion by SekiroZ** [pixiv/vroid-sdk-developers#74] [Q&A]
I've tried it on
https://www.pixiv.net/support?type=47&mode=inquiry&service=vroid-integrate

and
https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk&subject=VRoid

I need help, I am in the process of applying for the SDK.

![image](https://user-images.githubusercontent.com/58420788/233762752-73861378-024a-4586-b51b-c98f21b98f8f.png)

My contact email is

Sekiro@hybridtheory.io 

I would like a response from the official VRoid staff, either by email or community, telling me exactly what I need to do to register. Thank you!

**Comment by SekiroZ** [comment]
@nyamadan  hey help me

**Reply by nyamadan** [reply]
Were you unable to send your application through [this link](https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk&subject=VRoid)?

I will let the person in charge know about this.

**Reply by SekiroZ** [reply]
Tks

**Comment by SekiroZ** [comment]
nyamadan ***@***.***>於 2023年4月24日 週一，13:32寫道：

> Were you unable to send your application through this link
> <https://www.pixiv.net/support?mode=inquiry&type=43&service=vroid-sdk&subject=VRoid>
> ?
>
> I will let the person in charge know about this.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pixiv/vroid-sdk-developers/discussions/74#discussioncomment-5704104>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN5W4NBULMXUUFF5FIFHUZTXCYF75ANCNFSM6AAAAAAXHSUGZI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

Yes，tks
